### PR TITLE
Autohooks now uses getcwd instead of os.environ['PWD']

### PR DESCRIPTION
**Pull request by y0urself**
**What**:

* Using `getcwd` instead of `os.environ['PWD']`

**Why**:

* to fix Windows (?)

**How**:

**Checklist**:

- [ ] Tests
- [ ] [CHANGELOG](https://github.com/greenbone/autohooks/blob/master/CHANGELOG.md) Entry
- [ ] Documentation

### CODE REVIEW

**Comment by y0urself** [comment]
fix for #77 (?)

File: autohooks/utils.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def is_project_root(path: Path) -> bool:
 
 def get_project_root_path(path: Path = None) -> Path:
     if path is None:
-        path = Path(os.environ['PWD'])
+        path = Path(os.getcwd())
```
Comment (bjoernricks) [inline autohooks/utils.py:85]:
```suggestion
        path = Path.cwd()
```

**Review by bjoernricks — CHANGES_REQUESTED** [review]
Just a small simplification

File: autohooks/utils.py (hunk 1 above)
Comment (y0urself) [reply autohooks/utils.py:85]: Sure. Pathlib is even better ...

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**